### PR TITLE
Fix mislocated tooltips when UTF-8 characters are present (workaround lablgtk bug)

### LINKED ISSUE #19136: Tooltips are mislocated in CoqIDE when multibyte UTF-8 characters are present
### Description of the problem

image

The tooltip should be on "x", not on the period.

### Small Coq file to reproduce the bug

```coq
Definition α := x.
```

### Version of Coq where this bug occurs

8.19

### Last version of Coq where the bug did not occur

_No response_

### PR DESCRIPTION
**Pull request by jfehrle**
Fixes / closes #19136

This is a partial backport of #19040.

- [x] Added **changelog**.
- [ ] Added / updated **documentation**.

### CODE REVIEW

**Comment by jfehrle** [comment]
@coqbot: run full ci

File: ide/coqide/ideutils.ml [hunk 1]
```diff
@@ -174,6 +174,19 @@ let ulen uni_ch =
   else if uni_ch < 0x10000 then 3
   else 4
 
+(* workaround for lablgtk bug up to version 2.18.13 *)
```
Comment (ejgallego) [inline ide/coqide/ideutils.ml:177]: The version number doesn't seem right to me folks.

Comment (jfehrle) [reply ide/coqide/ideutils.ml:177]: @herbelin?

Comment (jfehrle) [inline ide/coqide/ideutils.ml:177]:
```suggestion
(* workaround for lablgtk bug up to version 3.1.4 *)
```

Comment (jfehrle) [reply ide/coqide/ideutils.ml]: It's 𝜋:  3.1.4

**Comment by SkySkimmer** [comment]
@coqbot run full ci

**Pull request merged**